### ISSUE TITLE
Accessibility: don't use font-size: 0

### ISSUE THREAD

**Issue by afercia** [voldemortensen/menu-customizer#92]
In some places, for example the "Add Menu Item" buttons or the "Toggle" buttons, font-size: 0 is used to hide the text. Unfortunately, this hides the text also from some assistive technologies. The text is just not announces, it says just "blank" (at least with Firefox+NVDA).
Would recommend to use `screen-reader-text` instead.
